packages/ui-extensions-dev-console/src/sections/Extensions/components/AppHomeRow/AppHomeRow.test.tsx

```diff
@@ -53,4 +53,40 @@ describe('<AppHomeRow/>', () => {
 
     expect(container).toContainReactComponent(QRCodeModal, {code: undefined})
   })
+
+  test("renders a <PreviewLink/> with the resource url set to the app's handle if the surface has been set to 'admin'", () => {
+    const appState = {
+      app: {url: 'mock.url', title: 'Mock App Title', handle: 'my-app-handle'},
+    }
+    const container = render(<AppHomeRow />, withProviders(DefaultProviders), {
+      state: appState,
+      client: {options: {surface: 'admin'}},
+    })
+
+    expect(container).toContainReactComponent(PreviewLink, {resourceUrl: '/apps/my-app-handle'})
+  })
+
+  test("renders a <PreviewLink/> with the resource url set to the app's handle if the surface has been set to 'admin'", () => {
+    const appState = {
+      app: {url: 'mock.url', title: 'Mock App Title', handle: 'my-app-handle'},
+    }
+    const container = render(<AppHomeRow />, withProviders(DefaultProviders), {
+      state: appState,
+      client: {options: {surface: 'admin'}},
+    })
+
+    expect(container).toContainReactComponent(PreviewLink, {resourceUrl: '/admin/apps/my-app-handle'})
```

Comment:
Isn't this test exactly the same as the one above, but the expected resourceUrl is different? 🤔

Reply:
Oops, sorry that's an accidental change - the second one is correct. It's concerning that the first test passes though. I'll look into it

Reply:
Actually, the test _was_ failing as expect! I'll clean it up